SwSpotify/__init__.py

```diff
@@ -9,9 +9,12 @@ class WebData:
 
     @staticmethod
     def set_song(data):
-        WebData.track = data['title']
-        WebData.artist = data['artist']
-        WebData.playState = data['playState']
+        try:
+            WebData.track = data['title']
+            WebData.artist = data['artist']
+            WebData.playState = data['playState']
+        except KeyError:
+            WebData.track = None
```

Comment:
You set just the track to None I assume just because that's what is used to check if SpotifyPaused should be raised or not?

Reply:
SpotifyNotRunning exactly in `get_info_web`

Reply:
Yeah, changed to SpotifyPaused

Reply:
Maybe SpotifyClosed makes more sense because it really isn't paused?
Sadly we don't provide a SpotifyInLimbo error lol

Reply:
lol, sure

Reply:
done